CMake fails on unknown policy

#### Description of Problem:
CMake fails on unknown policy.
This happened on our Jenkins when building #1093 
https://jenkins.open-scap.org/job/openscap-master-pull-requests/267/console

#### OpenSCAP Version:
master

#### Operating System & Version:
RHEL 7 on our Jenkins

#### Steps to Reproduce:

1.  `cd build`
2.  `cmake ..`
3. 


#### Actual Results:
```
CMake Error at CMakeLists.txt:467 (cmake_policy):
  Policy "CMP0063" is not known to this version of CMake.
```


#### Expected Results:
no fail

#### Additional Information / Debugging Steps:

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -464,7 +464,9 @@ include_directories(
 
 # Honor visibility properties for all target types
 # Run "cmake --help-policy CMP0063" for policy details
-cmake_policy(SET CMP0063 NEW)
+if (POLICY CMP0063)
+	cmake_policy(SET CMP0063 NEW)
+endif()
 
 function(set_oscap_generic_properties TARGET_OBJECT)
 	set_target_properties(${TARGET_OBJECT} PROPERTIES

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -466,6 +466,8 @@ include_directories(
 # Run "cmake --help-policy CMP0063" for policy details
 if (POLICY CMP0063)
 	cmake_policy(SET CMP0063 NEW)
+else()
+	message(WARNING "It is not possible to correctly set symbol visibility in object files with your version of CMake. We recommend using CMake 3.3 or newer.")
 endif()
 
 function(set_oscap_generic_properties TARGET_OBJECT)

